src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -29,7 +29,7 @@
     "suggest": {
         "symfony/yaml": "",
         "symfony/config": "",
-        "symfony/expression-language": "",
+        "symfony/expression-language": "For using the DependencyInjection YAML/XML file loaders and PHP/YAML/XML dumpers",
```

Comment:
This isn't right, you can use the loaders without having the expression language installed, you need it only if you use an expression in one of the configuration file. Maybe something like `"For using expressions in your service configurations"`.